packages/terra-menu/tests/wdio/menu-spec.js

```diff
@@ -6,21 +6,21 @@ Terra.describeViewports('Menu', ['medium'], () => {
   it('displays a default Menu', () => {
     browser.url('/raw/tests/cerner-terra-framework-docs/menu/menu/default-menu');
     $('#default-button').click();
-    $('[class*="content"][aria-modal="true"][role="dialog"]').isFocused();
+    expect($('li:first-child[role="menuitem"]').isFocused()).toBe(true);
```

Comment:
We can use `.toBeTruthy()` instead.

Reply:
https://github.com/cerner/terra-framework/pull/1626/commits/1d42b2c8065e80764079dcfc68dbabf55a810b07